packages/terra-core-docs/src/terra-dev-site/doc/icon/example/InformativeIconExamples.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import IconManufacturer from 'terra-icon/lib/icon/IconManufacturer';
+import IconAnnouncement from 'terra-icon/lib/icon/IconAnnouncement';
+
+import styles from './IconDocCommon.module.scss';
+
+const cx = classNames.bind(styles);
+const className = cx('example');
+
+const InformativeIconExamples = () => (
+  <>
+    <div className={className}>
+      <IconManufacturer title="Manufactured by" />
+      &nbsp;
+      2800 Rock Creek Pkwy Kansas City, MO 64117
+    </div>
+    <div className={className}>
+      <IconAnnouncement title="Announcement" height="2em" width="2em" />
```

Comment:
What is the reason for making the size of the Announcement icon 2em X 2em? It looks bigger compared to the Manufacture icon.

![icons](https://user-images.githubusercontent.com/22618655/157751026-cfbbf46b-4e88-4dd1-9688-f41784241877.png)

Reply:
not intentional. I'll make it the same size as the rest

Reply:
Fixed in f1f3766f9e4844615d37e6bcf519c905b648278a